### ISSUE TITLE
AssertJ containsEntry with json not working as expected

### ISSUE THREAD

**Issue by void-spark** [lukas-krecan/JsonUnit#361]
I would expect that a AssertJ containsEntry with json as value would work the same as inPath with isEqualTo.
But it fails instead.

```
import static net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson;
import static net.javacrumbs.jsonunit.assertj.JsonAssertions.json;
...

        assertThatJson(input,
                body -> body.isArray().hasSize(1),
// This check fails, I would expect it to work
                body -> body.inPath("[0]").isObject()
                        .containsEntry("allowedActions", json(
                             "{\n" +
                                "  \"approvable\" : true," +
                                "  \"rejectable\" : false" +
                              "}" )),
// This check on the other hand does not fail.
                body -> body.inPath("[0].allowedActions").isObject()
                         .isEqualTo(json(
                            "{\n" +
                                "  \"approvable\" : true," +
                                "  \"rejectable\" : false" +
                            "}" ))
        );
```

**Comment by lukas-krecan** [comment]
Hi, thanks for feedbck. It is indeed broken, will take a look at it.

**Comment by lukas-krecan** [comment]
Released a 2.26.0
